controls/gridsplitter/configuration.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Configuration
+page_title: .NET MAUI GridSplitter Documentation - Configuration
+description: Learn how to configure the Telerik UI for .NET MAUI GridSplitter resize direction and resize behavior.
+position: 3
+slug: gridsplitter-configuration
+---
+
+# .NET MAUI GridSplitter Styling
+
+The GridSplitter provides a set of properties for customizing its default behavior.
+
+## Parent Element
+
+You can use the `ParentLevel` (`int`) property to specify the level of the parent grid to resize
+
+## Resize Direction
+
+You can control whether the GridSplitter control resizes rows or columns by using the `ResizeDirection ` (`enum` of type `Telerik.Maui.Controls.RadGridSplitter.GridResizeDirection`) property. 
+
+The following options are supported:
+
+* (Default) `Auto`&mdash;Determines whether to resize rows or columns based on its alignment (based on the value of the `HorizontalAlignment` and `VerticalAlignment` properties) and width compared to height
+* `Columns`&mdash;Resizes columns when dragging the splitter.
+* `Rows`&mdash;Resize rows when dragging the splitter.
+
+## Resize Behavior
+
+You can control whether the GridSplitter control resizes rows or columns by using the `ResizeBehavior` (`enum` of type `Telerik.Maui.Controls.RadGridSplitter.GridResizeBehavior`) property. 
```

Comment:
The description of Resize Behavior and Resize Directions is the same. To add more value, describe what makes them different, don't use the same sentence.